app/src/main/java/protect/card_locker/CatimaAppCompatActivity.java

```diff
@@ -15,9 +16,13 @@
     SharedPreferences pref;
     HashMap<String, Integer> supportedThemes;
 
+    // store system locale for Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
```

Comment:
You say `>=` here, but in LoyaltyCardEditActivity this is used in the `<` version of the if loop. Should this comment be `<` instead of `>=`?